src/ui/components/NotificationGroup.tsx

```diff
@@ -2,11 +2,10 @@ import * as React from 'react';
 import { NotificationType } from './NotificationType';
 import { INotification } from '../models/INotification';
 import { INotificationGroupProps } from '../models/INotificationGroupProps';
-import { ISenderInfo } from '../../provider/Models/ISenderInfo';
-import { Fin } from '../../fin';
 import { eGroupMethod } from './App';
 
-declare var fin: Fin;
+import {NotificationCenterAPI} from '../NotificationCenterAPI';
+declare var window: Window&{notifications: NotificationCenterAPI};
```

Comment:
Do we want the API available on the window?

Reply:
we switched to using real types so can't really "extend" fin anymore.

Reply:
I will add an "openfin" in there so it's `window.openfin.notifications`